src/routes/Signup/Signup.scss

```diff
@@ -0,0 +1,214 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins');
```

Comment:
@alokesh985 What is the need to add this here?

Reply:
The Poppins font was not loading initially.
Now, it's working fine. Removing from here. Adding to global file.